src/pybind/Makefile

```diff
@@ -114,14 +114,18 @@ clean:
 	-rm -rf __pycache__
 	-rm -f $(CCFILES_OBJS)
 	-rm -f .depend.mk
+	-rm -f .mkl_env.mk
 
-test: all
+mkl:
+	-grep "LD_PRELOAD" ../../tools/env.sh > .mkl_env.mk
+
+test: mkl
```

Comment:
You can write
```
test: all mkl
```
here and no need to delete `all`

Reply:
resolved.

Reply:
I would prefer that you generate `mkl_env.mk` during compilation and use

```
ifdef MKLROOT
-include mkl_env.mk
endif
```

`MKLROOT` is defined in `kaldi.mk`
and remove the `mkl` rule in the Makefile.

Reply:
There's no need for a separate .mk file for the mkl stuff; kaldi.mk already contains all sorts of variables and there's no need to create a new file every time we add a new variable.   The `configure` script can just add whatever variables are needed to kaldi.mk.